spec/requests/bookings_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
+
+RSpec.describe "Bookings", type: :request do
+  describe "GET /index" do
+    pending "add some examples (or delete) #{__FILE__}"
+  end
+end
```

Comment:
Can you move this file inside the spec/controllers folder please and rename it bookings_controller_spec.rb please ?

While you're at it do you mind trying to do a few tests ?
You can take some inspiration from the existing events_controller_spec.rb 🙂 
It'd be a great exercice for you and it'd make the app more robust